.eslintrc.js

```diff
@@ -3,9 +3,7 @@ module.exports = {
   reportUnusedDisableDirectives: true,
   env: {
     node: true,
-  },
-  parserOptions: {
-    parser: '@babel/eslint-parser',
+    es2021: true,
```

Comment:
Nice, this would help Vite performance too. Any concern why we shouldn't go with `es2022`? We like having every modern syntax (at least in Stage 3) available.